website/docs/guides/0.8.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,144 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 0.8.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-080-upgrade-guide"
+description: |-
+    MongoDB Atlas Provider 0.8.0: Upgrade and Information Guide
+    
+---
+
+# MongoDB Atlas Provider v0.8.0: Upgrade and Information Guide
+
+Besides the bug fixes, improvements and enhancements listed in the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for 0.8.0 we want to call out some important changes and information that you should be aware of before upgrading to this newest provider.
+
+## Support for the switch from AWS credentials-based access (IAM user) to role-based access (IAM role) w/ Customer Key Management with AWS KMS
+
+MongoDB Atlas now supports IAM roles for use with AWS services.   Current users who had programmatically managed AWS KMS encryption keys for encryption at rest in the past 60 days were notified by email.  The MongoDB Atlas Provider v0.8.0 includes the support needed to use IAM roles (and transition from using IAM user).
+
+### Step 1: Configuring Cloud Provider Access
+
+If you programmatically create new Atlas projects and use AWS KMS encryption keys for encryption at rest or you are ready to switch an existing Atlas project using AWS KMS encryption keys for encryption at rest to using IAM users to roles, you will need to first configure the [cloud provider access](https://docs.atlas.mongodb.com/security/set-up-unified-aws-access) with https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_access.  The Atlas documentation for [cloud provider access](https://docs.atlas.mongodb.com/security/set-up-unified-aws-access) will walk you through the process in general.  
+
+Configuring cloud provider access with Terraform is a multi step process:
+
+1) Configure and apply the Atlas cloud_provide_access resource
+2) modify your AWS IAM role trust policy with AWS
+3) Add the IAM Assumed Role ARN to your Atlas resource definition
+4) Apply again to authorize.
+After this process you can then configure or update encryption at rest for your project.  For example:
```

Comment:
This line's showing up weird (gets swallowed into the list). Maybe add one extra new line after the list.